### DISCUSSION TITLE
Would love to see some kind of toast component for feedback.

### DISCUSSION THREAD

**Discussion by codethevoid** [heroui-inc/heroui#524] [Feedback]
Looking through the docs, it looks like there is no sort of toast component. I would love to see some kind of feedback component, I think it would really be great to use for form submissions, handling errors, and success messages, and other things of that nature.

**Comment by jrgarciadev** [comment]
Hey @rmthomas98 thanks for your recommendation, the `Toast` component is on our roadmap, stay tuned 👍🏻

**Reply by codethevoid** [reply]
That's awesome, I see you guys just rolled out the ```Dropdown``` component as well. Really loving it so far!

**Reply by jrgarciadev** [reply]
I'm glad you liked it ❤️
